moped-database/metadata/tables.yaml

```diff
@@ -2172,18 +2180,77 @@
             - notes
             - date_added
       retry_conf:
-        num_retries: 0
         interval_sec: 10
+        num_retries: 0
         timeout_sec: 60
       webhook_from_env: MOPED_API_EVENTS_URL
       headers:
         - name: MOPED_API_APIKEY
           value_from_env: MOPED_API_APIKEY
-        - value: activity_log
-          name: MOPED_API_EVENT_NAME
+        - name: MOPED_API_EVENT_NAME
+          value: activity_log
 - table:
+    name: moped_proj_personnel_roles
     schema: public
+  object_relationships:
+    - name: moped_proj_personnel
+      using:
+        foreign_key_constraint_on: project_personnel_id
+    - name: moped_project_role
+      using:
+        foreign_key_constraint_on: project_role_id
+  insert_permissions:
+    - role: moped-admin
+      permission:
+        check: {}
+        columns:
+          - is_deleted
+          - project_personnel_id
+          - project_role_id
+    - role: moped-editor
+      permission:
+        check: {}
+        columns:
+          - is_deleted
+          - project_personnel_id
+          - project_role_id
+  select_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - is_deleted
+          - id
+          - project_personnel_id
+          - project_role_id
+        filter: {}
+    - role: moped-editor
+      permission:
+        columns:
+          - is_deleted
+          - id
+          - project_personnel_id
+          - project_role_id
+        filter: {}
```

Comment:
are we going to skip setting permissions for moped-viewers since we aren't allowing anyone to be set as a viewer now?

Reply:
no—thank you for catching this! i added select permissions for viewers.